setup.py

```diff
@@ -10,7 +10,7 @@
     classifiers=["Programming Language :: Python :: 3 :: Only"],
     py_modules=["tap_stella"],
     install_requires=[
-        "singer-python>=5.9.0",
+        "singer-python>=5.7.3",
```

Comment:
is there any issue with me changing this? This is so that it'll work with our version of singer-python for the auto-selecting of streams

Reply:
It's unlikely to cause issues, esp if you've tested locally with that version. the only risks AFAIK are things like renamed or removed functions, new function signatures, etc.